### PR TITLE
Revert changes on the smtp server to fix nightly tests

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Revert changes on the smtp server to fix nightly tests (done by mistake)
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no tests needed
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26122)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

File: tests/UI/campaigns/utils/globals.js [hunk 1]
```diff
@@ -47,6 +47,6 @@ global.TAKE_SCREENSHOT_AFTER_FAIL = process.env.TAKE_SCREENSHOT_AFTER_FAIL || fa
 
 global.maildevConfig = {
   smtpPort: process.env.SMTP_PORT || '1025',
-  smtpServer: process.env.SMTP_SERVER || 'localhost',
+  smtpServer: process.env.SMTP_SERVER || '172.20.0.4',
```
Comment (PierreRambaud) [inline tests/UI/campaigns/utils/globals.js]: what's this IP? :thinking:

Comment (boubkerbribri) [reply tests/UI/campaigns/utils/globals.js]: If I'm not wrong, it's the IP shared between PrestaShop and tests Containers.
We start the maildev server on tests, and to make the shop able to communicate with it, we should use the ip and not localhost.
You could see the config [here](https://github.com/PrestaShop/PrestaShop/blob/develop/tests/UI/docker-compose.tests.yml#L3-L7).

Comment (PierreRambaud) [reply tests/UI/campaigns/utils/globals.js]: I recommend to keep the localhsot value and configure SMPT_SERVER here https://github.com/PrestaShop/PrestaShop/blob/develop/tests/UI/docker-compose.tests.yml#L20 instead

Comment (boubkerbribri) [reply tests/UI/campaigns/utils/globals.js]: done

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @boubkerbribri

**Pull request merged**